controllers/contentlibrary/utils/utils.go

```diff
@@ -65,8 +65,18 @@ func AddContentLibraryRefToAnnotation(to client.Object, ref *imgregv1a1.NameAndK
 	if annotations == nil {
 		annotations = map[string]string{}
 	}
-	annotations[v1alpha2.VMIContentLibRefAnnotation] = string(data)
+	annotations[vmopv1.VMIContentLibRefAnnotation] = string(data)
 	to.SetAnnotations(annotations)
 
 	return nil
 }
+
+// AddImageTypeToLabel adds the image type label to the given VMI or CVMI object.
+func AddImageTypeToLabel(to client.Object, imageType string) {
+	labels := to.GetLabels()
+	if labels == nil {
+		labels = map[string]string{}
+	}
+	labels[vmopv1.VirtualMachineImageTypeLabel] = imageType
```

Comment:
Check that the label value is valid - there's other examples in this package.

Reply:
Thanks! Addressed in https://github.com/vmware-tanzu/vm-operator/compare/83759cd66b4c75f0ab73da4260a9cb00ff0cda7d..719de5bea73dc46dae720b777067d51bfb2afe5b